### PR TITLE
Create card

### PR DESCRIPTION
**Pull request by JohannesMisch**

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,3 +1,67 @@
...
+export default function Home({ devices }) {
+  console.log(devices, Entertainment);
+
+  function handleImage(category) {
+    console.log(category);
+    if (category === "Entertainment") {
+      return (
+        <svg xmlns="http://www.w3.org/2000/svg" height="48" width="48">
+          <path d="M7.35 38q-1.9 0-2.975-1.275Q3.3 35.45 3.6 33.2L6 16.45q.4-2.65 2.475-4.55T13.2 10h21.65q2.65 0 4.725 1.9 2.075 1.9 2.475 4.55L44.4 33.2q.3 2.25-.775 3.525T40.65 38q-1.15 0-1.95-.375t-1.35-.925l-5.2-5.2h-16.3l-5.2 5.2q-.55.55-1.35.925T7.35 38Zm.9-3.2 6.3-6.3h18.9l6.3 6.3q.25.25.9.45.45 0 .675-.45.225-.45.125-.9l-2.4-16.95q-.25-1.75-1.475-2.85T34.85 13h-21.7q-1.5 0-2.725 1.1T8.95 16.95L6.55 33.9q-.1.45.125.9t.675.45q.35 0 .9-.45ZM35 26q.8 0 1.4-.6.6-.6.6-1.4 0-.8-.6-1.4-.6-.6-1.4-.6-.8 0-1.4.6-.6.6-.6 1.4 0 .8.6 1.4.6.6 1.4.6Zm-4.25-6.5q.8 0 1.4-.6.6-.6.6-1.4 0-.8-.6-1.4-.6-.6-1.4-.6-.8 0-1.4.6-.6.6-.6 1.4 0 .8.6 1.4.6.6 1.4.6ZM15 25.75h2.5V22h3.75v-2.5H17.5v-3.75H15v3.75h-3.75V22H15Zm9-1.65Z" />
+        </svg>
```
Comment (F-Kirchhoff) [inline pages/index.js]: You can move these svgs into their own icon components in a later pr.

File: pages/index.js [hunk 2]
```diff
@@ -1,3 +1,67 @@
...
 }
+const StyledCard = styled.li`
+  border: solid black 3px;
+`;
+const StyledList = styled.ul`
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  padding: 20px 10px 20px 10px;
```
Comment (F-Kirchhoff) [inline pages/index.js]: you can use the shorthand here: `padding: 20px 10px`

File: pages/index.js [hunk 3]
```diff
@@ -1,3 +1,67 @@
...
+`;
+const StyledIcon = styled.img``;
```
Comment (F-Kirchhoff) [inline pages/index.js]: Where do you use this component?

File: pages/index.js [hunk 4]
```diff
@@ -1,3 +1,67 @@
-export default function HomePage() {
-  return <h1>Hello from Next.js</h1>;
+import styled from "styled-components";
+import { ReactComponent as Entertainment } from "../assets/Entertainment.svg";
+
+export default function Home({ devices }) {
+  console.log(devices, Entertainment);
```
Comment (F-Kirchhoff) [inline pages/index.js]: no console.logs in the PR please 🕵️

File: pages/index.js [hunk 5]
```diff
@@ -1,3 +1,67 @@
...
+
+  function handleImage(category) {
+    console.log(category);
```
Comment (F-Kirchhoff) [inline pages/index.js]: 🕵️

File: pages/_app.js [hunk 6]
```diff
@@ -1,14 +1,23 @@
 import GlobalStyle from "@/styles";
 import Head from "next/head";
+import useLocalStorageState from "use-local-storage-state";
+import Devices from "../db.json";
 
 export default function App({ Component, pageProps }) {
+  const [devices, setDevices] = useLocalStorageState("devices", {
+    defaultValue: [...Devices],
```
Comment (F-Kirchhoff) [inline pages/_app.js]: you don't need to spread the devices here into a new array, and please use a lowercase name, Capital names are reserved for react components and classes.

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
Hey Johannes, I like your code a lot and found only small things like console logs. Have a look!

File: pages/index.js (hunk 4 above)
Comment (JohannesMisch) [reply pages/index.js]: Thank you, totally forgot them.

File: pages/index.js (hunk 3 above)
Comment (JohannesMisch) [reply pages/index.js]: I don't, it is a fragment of a long-forgotten past. :)

File: pages/_app.js (hunk 6 above)
Comment (JohannesMisch) [reply pages/_app.js]: fixed the Capital names, but if I don't spread the "Devices" I run into problems with the unique keys. :(

**Comment by Mistake91** [comment]
I see you made the changes Felix suggested, so for me it looks good:)

**Pull request merged**